packages/components/src/components/tag/tag.hbs

```diff
@@ -15,9 +15,9 @@
 
 <div role="list">
   {{#if filter}}
-  <span class="{{@root.prefix}}--tag {{@root.prefix}}--tag--filter" title="Clear filter" tabindex="0" role="listitem">
+  <button class="{{@root.prefix}}--tag {{@root.prefix}}--tag--filter" title="Clear filter" role="listitem">
```

Comment:
Does `role="listitem"` make sense if it's now a button? 🤔 Would we want to handle hbs updates in a separate PR?

Reply:
removed the outdated roles. it's a documentation change so I think it's ok for it to be contained in this PR